demo_web/src/main.rs

```diff
@@ -1,16 +1,61 @@
+use demo::Provider;
+use walkers::sources::{Attribution, TileSource};
+use walkers::TileId;
+
+#[derive(Default)]
+pub struct CustomSource {
```

Comment:
Indeed, but if we need to make a PR for every new source, it would be impossible to maintain

With a source provided by the user, it's simpler

I renamed `CustomSource` to `MySource`